### PR TITLE
feat(parser): ansible inventory

### PR DESCRIPTION
**Pull request by cx-henriqueAlvelos**
**Proposed Changes**
- Parser for Ansible Inventory
- Tests

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Review by pereiramarco011 — CHANGES_REQUESTED** [review]
The template query should be removed as it is simply an example to test the new feature and not a real vulnerability.

Along with that I have added a few nitpick comments that are optional 😄

**Review by pereiramarco011 — APPROVED** [review]
LGTM 😄

File: pkg/parser/ansible/ini/comments/comments_test.go [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+				"ansible_host=0.0.0.0",
+			},
+			want: []int{0, 1},
+		},
+	}
+
+	for i, tt := range tests {
+		t.Run("Test number "+strconv.Itoa(i), func(t *testing.T) {
+			if i == 3 {
+				fmt.Println("olaola")
```
Comment (cxMiguelSilva) [inline pkg/parser/ansible/ini/comments/comments_test.go:130]:
```suggestion
```
Remove test Print

**Review by cxMiguelSilva — CHANGES_REQUESTED** [review]
Hi @cx-henriqueAlvelos, 
The proposed changes to add the ini parser  LGTM 🚀
Please remove the Test print on the unit test file.

**Review by cxMiguelSilva — DISMISSED** [review]

**Review by pereiramarco011 — APPROVED** [review]
LGTM 😄

**Pull request merged**